etc/kayobe/kolla-image-tags.yml

```diff
@@ -5,19 +5,19 @@
 kolla_image_tags:
   openstack:
     rocky-9: zed-rocky-9-20230921T153510
-    ubuntu-jammy: zed-ubuntu-jammy-20230921T153510
+    ubuntu-jammy: zed-ubuntu-jammy-20240129T151534
   bifrost:
     rocky-9: zed-rocky-9-20230927T142529
-    ubuntu-jammy: zed-ubuntu-jammy-20231101T132522
+    ubuntu-jammy: zed-ubuntu-jammy-20240129T151534
```

Comment:
I don't think you need to define any of the tags that are the same as the default openstack tag

Reply:
Should I remove all `ubuntu-jammy` entries that are same to default tag and leave only `rocky-9` tags?

Reply:
Yes

Reply:
Actually, it may be worth waiting until Will's PR can merge, since it will conflict with this one
https://github.com/stackhpc/stackhpc-kayobe-config/pull/910

Reply:
It's done